server/channels/api4/system.go

```diff
@@ -194,6 +195,13 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	s["ActiveSearchBackend"] = c.App.ActiveSearchBackend()
 
+	// Checking if mattermost is running as root
+	rootStatusKey := "root_status"
+	s[rootStatusKey] = "false"
```

Comment:
While I see that it is hard to add a test for the case that MM is run as root, I wonder if it's reasonable to ensure that `"root_status"` is false?

Reply:
You can reuse the existing `TestGetPing`test in `server/channels/api4/system_test.go` and add your own check for `"root_status"`.